src/endpoints/authentication/impl.cpp

```diff
@@ -180,8 +173,7 @@ namespace irods::http::handler
 		log::debug("{}: Authorization value (trimmed): [{}]", __func__, authorization);
 
 		constexpr auto max_creds_size = 128;
-		unsigned long size = max_creds_size;
-		//std::vector<std::uint8_t> creds(size);
+        std::uint64_t size{max_creds_size};
```

Comment:
indentation

Reply:
Looks like spaces were used instead of tabs

Reply:
I'll be running clang-format, deferring formatting discussions to this thread(?): https://github.com/irods/irods_client_http_api/pull/75#discussion_r1334508464